### ISSUE TITLE
vulnerability CVE-2022-30065

### ISSUE THREAD

**Issue by sw-fox** [nginxinc/docker-nginx#685]
we got the security finding CVE-2022-30065 on the image nginx:1.23.0-alpine
is there a fix planned?

**Comment by thresheek** [comment]
Hi @sw-fox !

It's already fixed, please pull the image again.

**Comment by sw-fox** [comment]
I pulled the image half an hour ago. Is this the right one? There was no new tag, or patch version. The vulnerability is still found.
```
docker pull nginx:1.23.0-alpine
1.23.0-alpine: Pulling from library/nginx
Digest: sha256:4a846cc240449c53c8ae24269ba6bcaee5167d8ad75cd2a8d8ba422b7c726979
Status: Image is up to date for nginx:1.23.0-alpine
```

**Comment by thresheek** [comment]
Yes, that's the correct image.  It has busybox-1.35.0-r15, which contains the fix for the aforementioned CVE.

**Comment by thresheek** [comment]
Check `apk info -v | grep busybox`.

**Comment by sw-fox** [comment]
this seems to be the newer version
```
$ apk info -v | grep busybox
busybox-1.35.0-r15
```

**Comment by felipeconti** [comment]
Trivy is showing vulnerability is still found on version of busybox `busybox-1.35.0-r15`.

![image](https://user-images.githubusercontent.com/1789910/179781865-705b21e9-1c45-46b1-87f4-4125ae02a982.png)

**Comment by thresheek** [comment]
You need to fix your security scanner then, since it's fixed in r15: https://git.alpinelinux.org/aports/commit/main/busybox?h=3.16-stable&id=e58f0e341603e9facc1e675722367174b45e0d0c

**Comment by sw-fox** [comment]
@thresheek in your link is only version r17 mentioned, I couldn't find any information about version r15?
![grafik](https://user-images.githubusercontent.com/56607198/179798904-09b738e9-e2f8-4847-8074-5aa25aff1a10.png)

**Comment by thresheek** [comment]
That's because it was cherry-picked from the edge, which had r17 at this point.

The actual package version is in the beginning of APKBUILD there.

**Comment by felipeconti** [comment]
@thresheek related this link https://security.alpinelinux.org/vuln/CVE-2022-30065 `1.35.0-r15` are possibly vulnerable and fixed on `1.35.0-r18`

**Comment by thresheek** [comment]
Thanks @felipeconti ! It currently says that it's fixed on that page as well.

**Comment by sw-fox** [comment]
our build does an `apk upgrade && apk update` and since tonight this seems to fix it.
Still the same baseimage (we use a copy in our own internal registry)
